src/three/controls/GlobeControls.js

```diff
@@ -152,7 +153,19 @@ export class GlobeControls extends EnvironmentControls {
 		// update the projection matrix
 		const largestDistance = Math.max( ...ellipsoid.radius );
 		camera.near = Math.max( 1, distanceToCenter - largestDistance * 1.25 );
-		camera.far = distanceToCenter + largestDistance + 0.1;
+
+		// update the far plane to the horizon distance
+		ellipsoid.getPositionToCartographic( camera.position, _vec );
```

Comment:
This function call doesn't look right? The camera position value needs to be transformed into the local "tilesGroup" space, first. This function also should take an empty object since it assigns values to "lat" and "lon". Right now this is modifying a "Vector3" object.

Reply:
yes nice catch, I guess the difference / influence of that variable was minimal I don't notice any change, but yes it's absolutely the right way to do it.